scripts/makeArtifactList.js

```diff
@@ -46,7 +46,7 @@ if (isCheckMode) {
 }
 
 function getFilesAddedByCI() {
-  const platforms = ["darwin", "darwinarm64", "linux", "win32"];
+  const platforms = ["darwin", "darwinarm64", "linux", "linuxarm64", "win32"];
```

Comment:
I guess this is a tangentially related fix.

Reply:
Yes, I noticed this when regenerating the artifact list.